CONTRIBUTING.md

```diff
@@ -1,36 +1,36 @@
-# Contributor's Guide
+# Contributor's guide
 
 We welcome pull requests from cmd2 users and seasoned Python developers alike! Follow these steps to contribute:
 
-1. Find an issue that needs assistance by searching for the [Help Wanted](https://github.com/python-cmd2/cmd2/labels/help%20wanted) tag.
+1. Find an issue that needs assistance by searching for the [Help Wanted](https://github.com/python-cmd2/cmd2/labels/help%20wanted) tag
 
-2. Let us know you are working on it by posting a comment on the issue.
+2. Let us know you're working on it by posting a comment on the issue
 
-3. Follow the [Contribution Guidelines](#contribution-guidelines) to start working on the issue.
+3. Follow the [Contribution guidelines](#contribution-guidelines) to start working on the issue
 
 Remember to feel free to ask for help by leaving a comment within the Issue.
 
-Working on your first Pull Request? You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
+Working on your first pull request? You can learn how from this *free* series [How to Contribute to an Open Source Project on GitHub](https://egghead.io/series/how-to-contribute-to-an-open-source-project-on-github).
 
 ###### If you've found a bug that is not on the board, [follow these steps](#found-a-bug).
```

Comment:
This link is (and was already) broken.

Reply:
Good catch!  My intent was to add a "Found a Bug?" section towards the end of this contributing guide and it looks like I forgot to do that.

I was intending it to look something like the following:

### Found a bug?

If you think you've found a bug, please first read through the open [Issues](https://github.com/python-cmd2/cmd2/issues). If you're confident it's a new bug, go ahead and create a new GitHub issue. Be sure to include as much information as possible so we can reproduce the bug.  At a minimum, please state the following:

* ``cmd2`` version
* Python version
* OS name and version